src/common/state/redis.c

```diff
@@ -153,22 +163,34 @@ db_handle *db_connect(const char *address,
     freeReplyObject(reply);
     reply = redisCommand(context, "MULTI");
     freeReplyObject(reply);
-    reply = redisCommand(
-        context,
-        "HMSET db_clients:%b client_type %s address %s:%d db_client_id %b",
-        (char *) client.id, sizeof(client.id), client_type, client_addr,
-        client_port, (char *) client.id, sizeof(client.id));
+    reply = redisCommand(context,
+                         "HMSET db_clients:%b client_type %s address %s:%d "
+                         "db_client_id %b aux_address %s",
+                         (char *) client.id, sizeof(client.id), client_type,
+                         client_addr, client_port, (char *) client.id,
+                         sizeof(client.id), aux_address);
+    CHECK_REDIS_CONNECT(redisContext, context,
```

Comment:
this should be just CHECKM(reply != NULL, "db_connect failed on HMSET");
The CHECK_REDIS_CONNECT is only for redisConnect commands, not for redisCommand